### DISCUSSION TITLE
Welcome to Abbreve Discussions!

### DISCUSSION THREAD

**Discussion by Njong392** [Njong392/Abbreve#160] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ajimbong** [comment]
I've been using Abbreve these days and it's absolutely amazing.
I no longer have to spoil the mood of a conversation by always asking people what they mean.

**Reply by Njong392** [reply]
> I've been using Abbreve these days and it's absolutely amazing. I no longer have to spoil the mood of a conversation by always asking people what they mean.

Thank you very much for the feedback! I appreciate it very much, and I am happy Abbreve is helping someone haha.
